src/Engine/ProtoCore/CodeGen.cs

```diff
@@ -496,6 +496,63 @@ protected ProtoCore.DSASM.DebugInfo GetDebugObject(int line, int col, int eline,
             ProtoCore.AST.Node setterArgument = null
             );
 
+        //
+        //  proc AutoGenerateUpdateArgumentReference(node)
```

Comment:
My suggestion would be move this comment to the body of function and use xml documentation here to indicate the usage of this function.